packages/monitor-v2/src/monitor-polymarket/MonitorProposalsOrderBook.ts

```diff
@@ -50,7 +51,8 @@ export async function monitorTransactionsProposedOrderBook(
 async function processMarketProposal(market: OptimisticPriceRequest, params: MonitoringParams, logger: typeof Logger) {
   const networker = new Networker(logger);
   const questionID = calculatePolymarketQuestionID(market.ancillaryData);
-  const polymarketInfo = await getPolymarketMarketInformation(params, questionID);
+  // set this to retry twice and wait 5 seconds between retries.
+  const polymarketInfo = await retryAsync(() => getPolymarketMarketInformation(params, questionID), 2, 5000);
```

Comment:
It could be useful to set these hardcoded values (retries and delayMS) as environment variables, with their default values being the current hardcoded values

https://github.com/UMAprotocol/protocol/blob/be2e91dd192e9368614f41926e8ac6e4090719b2/packages/monitor-v2/src/monitor-polymarket/common.ts#L345

Reply:
yep i agree, was being lazy, but i can do that